src/transformers/generation_utils.py

```diff
@@ -407,6 +474,13 @@ def generate(
                 conditioned on the previously generated tokens :obj:`inputs_ids` and the batch ID :obj:`batch_id`. This
                 argument is useful for constrained generation conditioned on the prefix, as described in
                 `Autoregressive Entity Retrieval <https://arxiv.org/abs/2010.00904>`__.
+            output_attentions (:obj:`bool`, `optional`, defaults to `False`):
```

Comment:
We leave the `defaults to "False"` here because `model.config.output_attentions` defaults to `False`